src/creatures/players/player.cpp

```diff
@@ -1162,56 +1141,39 @@ ReturnValue Player::rewardChestCollect(const Container* fromCorpse /* = nullptr*
 	auto rewardCount = rewardItemsVector.size();
 	bool fallbackConsumed = false;
 	Item* fallbackItem = getInventoryItem(CONST_SLOT_BACKPACK);
-	int32_t movedMoney = 0;
+	int32_t movedRewardMoney = 0;
 	for (auto item : rewardItemsVector) {
 		if (uint32_t worth = item->getWorth(); worth > 0) {
```

Comment:
Wouldn't `internalQuickLootItem` do the right thing here? Or do you want to force it to go to the bank even if autoBank is off?

Reply:
Yes, that was exactly the idea. It's just that every 100 gold coins is an item request, which "weighs" a little. The way I put it, it's less expensive, since it doesn't do all the add/removePostNotification checks and all that stuff, so this makes the function more optimized and less expensive, in my opinion. And monsters tend to drop a very large amount of gold.